packages/shared/src/hooks/source/useSourceActionsNotify.ts

```diff
@@ -9,60 +9,62 @@ import { useNotificationPreferenceToggle } from '../notifications';
 import { useToastNotification } from '../useToastNotification';
 
 export type UseSourceSubscriptionProps = {
-  source: Pick<Source, 'id'>;
+  source: Pick<Source, 'id'> | Source;
 };
 
 export type UseSourceSubscription = {
-  isSubscribed: boolean;
+  haveNotifications: boolean;
   isReady: boolean;
-  onSubscribe: () => Promise<void>;
+  onNotify: () => Promise<void>;
 };
 
-export const useSourceSubscription = ({
+export const useSourceActionsNotify = ({
   source,
 }: UseSourceSubscriptionProps): UseSourceSubscription => {
   const { logEvent } = useLogContext();
   const { isLoggedIn, showLogin } = useAuthContext();
   const { displayToast } = useToastNotification();
-  const { isSubscribed, isReady, onToggle } = useNotificationPreferenceToggle({
-    params: source?.id
-      ? {
-          notificationType: NotificationType.SourcePostAdded,
-          referenceId: source?.id,
-        }
-      : undefined,
-  });
+  const { haveNotifications, isReady, onToggle } =
+    useNotificationPreferenceToggle({
+      params: source?.id
+        ? {
+            notificationType: NotificationType.SourcePostAdded,
+            referenceId: source?.id,
+          }
+        : undefined,
+    });
 
-  const onSubscribe = useCallback(async () => {
+  const onNotify = useCallback(async () => {
     if (!source?.id) {
       return;
     }
 
     if (!isLoggedIn) {
       showLogin({ trigger: AuthTriggers.SourceSubscribe });
-
       return;
     }
 
-    const result = await onToggle();
+    const notifications = await onToggle();
 
     logEvent({
-      event_name: result.isSubscribed
+      event_name: notifications.isSubscribed
         ? LogEvent.SubscribeSource
         : LogEvent.UnsubscribeSource,
       target_id: source.id,
     });
 
+    const displayName = 'name' in source ? source.name : source?.id;
+
     displayToast(
-      result.isSubscribed
-        ? '✅ You are now subscribed'
-        : '⛔️ You are now unsubscribed',
+      notifications.isSubscribed
+        ? `✅ You'll get notified every time ${displayName} posts`
+        : `⛔️ You'll no longer get notified about ${displayName} posts`,
```

Comment:
Is this copy change intended even without experiment?

Reply:
I kept because it make sense and clarify whats going on, but maybe is more a part of the experiment and should revert :\

Reply:
Yeah, we either move this behind experiment, or we check with product in this case for copy change.